examples/src/main/java/io/numaproj/numaflow/examples/function/reduce/sum/SumFactory.java

```diff
@@ -0,0 +1,21 @@
+package io.numaproj.numaflow.examples.function.reduce.sum;
+
+import io.numaproj.numaflow.function.FunctionServer;
+import io.numaproj.numaflow.function.reduce.ReducerFactory;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+
+@Slf4j
+public class SumFactory extends ReducerFactory<SumFunction> {
+
+    public static void main(String[] args) throws IOException {
+        log.info("sum udf was invoked");
+        new FunctionServer().registerReducer(new SumFactory()).start();
+    }
+
+    @Override
+    public SumFunction createReducer() {
+        return new SumFunction();
```

Comment:
In CountFactory.java, we use `public static class EvenOddCounter extends Reducer` to declare the Reducer function. Consider following the same pattern by moving the SumFunction.java to within SumFactory, to be consistent.

Reply:
The intent was to show different ways of writing a Reducer.